### PR TITLE
MONIT-35754: The proxy install.sh file should be updated to support CSP credentials for Mac os.

### PR DESCRIPTION
**Pull request by Norayr25**
…SP credentials for Mac os.

### CODE REVIEW

File: conf/wavefront.conf [hunk 1]
```diff
@@ -18,7 +18,27 @@ hostname=myHost
 # 1. Click the gear icon at the top right in the Wavefront UI.
 # 2. Click your account name (usually your email)
 # 3. Click *API access*.
-token=TOKEN_HERE
+token=WAVEFRONT_API_TOKEN_HERE
```
Comment (laullon) [inline conf/wavefront.conf]: this should be commented (and un-commented during install)

Comment (Norayr25) [reply conf/wavefront.conf]: > this should be commented (and un-commented during install)

Fixed

**Review by laullon — APPROVED** [review]
Initially LGTM, but we need to test it and merge only on release day.

so, DO NOT MERGE

**Comment by Norayr25** [comment]
> Initially LGTM, but we need to test it and merge only on release day.
> 
> so, DO NOT MERGE

okay

**Pull request merged**